iceberg/iceberg-handler/src/test/results/positive/write_iceberg_branch.q.out

```diff
@@ -237,19 +237,19 @@ STAGE PLANS:
                   alias: ice01
                   filterExpr: (a = 22) (type: boolean)
                   Snapshot ref: branch_test1
-                  Statistics: Num rows: 3 Data size: 291 Basic stats: COMPLETE Column stats: COMPLETE
+                  Statistics: Num rows: 5 Data size: 485 Basic stats: COMPLETE Column stats: COMPLETE
```

Comment:
Before this PR, we always get` row count` of  branch/tag/timetravel by the _**current snapshot**_ summary, which is not right.